WeakAuras/SubRegionTypes/Tick.lua

```diff
@@ -314,6 +321,15 @@ local function modify(parent, region, parentData, data, first)
   region.automatic_length = data.automatic_length
   region.tick_thickness = data.tick_thickness
   region.tick_length = data.tick_length
+  region.use_texture = data.use_texture
+  region.tick_texture = data.tick_texture
+  region.tick_blend_mode = data.tick_blend_mode
+
+
+  if data.use_texture then
+    WeakAuras.SetTextureOrAtlas(region.texture, data.tick_texture)
```

Comment:
I think we should default to CLAMPTOBLACKADDITIVE as the wrap mode (3. and 4. argument)